tests/functional/test_home.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.home import HomePage
+
+
+@pytest.mark.nondestructive
+def test_promo_grid_is_displayed(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.is_promo_grid_displayed
+
+
+@pytest.mark.nondestructive
+def test_promos_are_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.expected_promos == 16
+
+
+@pytest.mark.nondestructive
+def test_tweet_is_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.is_tweet_promo_present
```

Comment:
Did you say there are cases where this is not displayed? I noticed it wasn't there for en-GB. If that's expected then we can instantiate the page object with a specific locale for a negative test.